components/layout/AppMobileNavbar.vue

```diff
@@ -18,22 +18,24 @@
             }}</NuxtLink>
           </li>
           <li>
-            <NuxtLink class="font-bold block px-4 py-3" to="/projects?first">{{
-              $t('navbar.navbarProjects')
-            }}</NuxtLink>
+            <span
```

Comment:
nitpick(blocking): since the redirect is now an click we can loose the span div and use the li element directly, same for the other ones

Reply:
Good point